api/swagger-spec/v1.json

```diff
@@ -5971,7 +5971,7 @@
         "allowMultiple": false
        },
        {
-        "type": "",
+        "type": "string",
```

Comment:
As commented on the other issue CSV is incredibly dangerous - I would much prefer to say that swagger is broken, and we need to update to 2.0.  I don't think CSV makes the problem better.